src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,79 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+/*public class ArmSubsystem extends SubsystemBase {
```

Comment:
Why the big comment?

Reply:
uhhh I did some work on the extension code from friday, but I never got a chance to finish it and there are some issues with it I need to fix.